gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/token/transfer.ts

```diff
@@ -20,28 +21,63 @@ export default class TransferToken extends SolanaCommand {
     this.require(!!args[0], 'Provide a token address')
   }
 
-  execute = async () => {
+  isValidTokenAccount = async (token: Token, account: PublicKey) => {
```

Comment:
This is a duplicate of declared `isAssociatedTokenAddressCreated` fn. Please consider extracting to one canonical fn.

Reply:
Extracted it to a token/utils